libirmin.opam

```
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "C bindings for irmin"
+description: "C bindings for irmin using Ctypes inverted stubs"
+maintainer: ["zachshipko@gmail.com"]
+authors: ["Zach Shipko"]
+license: "ISC"
+homepage: "https://github.com/mirage/irmin"
+bug-reports: "https://github.com/mirage/irmin/issues"
+depends: [
+  "dune" {> "2.0"}
```

```suggestion
  "dune" {> "2.7"}
```

Minimal version needs to match the language version in `dune-project` (not that it matters w/ an `=version` constraint on `irmin-pack` anyway).